src/elements/content-sidebar/stories/tests/BoxAISidebar-visual.stories.tsx

```diff
@@ -1,19 +1,49 @@
-import { expect, userEvent, within } from '@storybook/test';
+import {expect, fn, screen, userEvent, waitFor, within} from '@storybook/test';
 import { type StoryObj } from '@storybook/react';
+import {AgentType} from '@box/box-ai-agent-selector';
 import ContentSidebar from '../../ContentSidebar';
 import BoxAISidebar from '../../BoxAISidebar';
 
 const mockFeatures = {
     'boxai.sidebar.enabled': true,
 };
 
+export const mockAgents: AgentType[] = [
+    {
+        id: '1',
+        name: 'Agent 1',
+        description: 'This is the default agent',
+        isEnterpriseDefault: true,
+        imageURL: 'https://cdn01.boxcdn.net/_assets/img/favicons/favicon-32x32.png',
+    },
+    {
+        id: '2',
+        name: 'Agent 2',
+        description: 'This agent has a different description',
+        isEnterpriseDefault: false,
+    },
+    {
+        id: '3',
+        name: 'Agent 3',
+        isEnterpriseDefault: false,
+        ask: { foo: 'foobar' },
+    },
+    {
+        id: '4',
+        name: 'Agent 4',
+        description: 'This is agent 4',
+        imageURL: 'https://cdn01.boxcdn.net/_assets/img/favicons/favicon-32x32.png',
+    },
+];
+
 export default {
```

Comment:
We need to mock the api request, otherwise the snapshots wont render correctly. 

something like this need to be added to the default or to the stories directly https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/stories/ContentSidebar.stories.js#L36